ndt7/download/receiver/receiver.go

```diff
@@ -1,63 +0,0 @@
-// Package receiver implements the counter-flow messages receiver.
```

Comment:
Renamed (unclear to me why it's not shown as a renamed and slightly changed file)